clang/test/Driver/cxx23-modules-print-library-module-manifest-path.c

```diff
@@ -0,0 +1,15 @@
+// Test that -print-library-module-manifest-path finds the correct file.
+
+// RUN: %clang -print-library-module-manifest-path \
+// RUN:     -stdlib=libc++ \
+// RUN:     --sysroot=%S/Inputs/cxx23_modules \
+// RUN:     --target=x86_64-linux-gnu 2>&1 \
+// RUN:   | FileCheck --check-prefix=CHECK-LIBCXX %s
+// CHECK-LIBCXX: module: ={{.*}}/Inputs/cxx23_modules/usr/lib/x86_64-linux-gnu/modules.json
+
```

Comment:
Is there a path for libc++ like `--gcc-toolchain=` so that we can choose the different locations for libstdc++?

Reply:
not sure, I've never used --gcc-toolchain either.